src/main/java/com/wire/bots/roman/MessageHandler.java

```diff
@@ -356,6 +383,7 @@ private boolean send(OutgoingMessage message) {
             }
         } catch (Exception e) {
             Logger.error("MessageHandler.send: bot: %s, provider: %s,  error %s", message.botId, providerId, e);
+            e.printStackTrace();
```

Comment:
We need to find a bit different way how to do this, because this one does not get caught by the logger and then it is not in Kibana -> I'll have a look how we should handle that

Reply:
solved here https://github.com/wireapp/xenon/pull/4